py3status/module.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, module, user_modules, py3_wrapper, instance=None):
         self.module_name = module.split(" ")[0]
         self.new_update = False
         self.nagged = False
+        self.on_error = "show"
```

Comment:
Following style, `self.on_error = None` because the default value will be set below.